ivy_tests/test_ivy/test_frontends/test_tensorflow/test_math.py

```diff
@@ -2522,6 +2522,39 @@ def test_tensorflow_scalar_mul(
     )
 
 
+# segment_min
+@handle_frontend_test(
+    fn_tree="tensorflow.math.segment_min",
+    data=helpers.array_values(
+        dtype=ivy.float64, shape=(5, 6), min_value=1, max_value=9
+    ),
+    segment_ids=helpers.array_values(
+        dtype=ivy.int64, shape=(5,), min_value=0, max_value=4
+    ),
```

Comment:
Can we change these to use `dtype_and_x` like the other tests?

Reply:
done